### PR TITLE
retrace: Move stuff from retrace to util

### PR DESCRIPTION
**Pull request by mkutlak**
The goal of this commit is to free `retrace.py` from stuff that
can be placed elsewhere (`util.py`) and make it more structured.

Most of the moved functions in this commit are never used in `retrace.py`.

### CODE REVIEW

**Comment by mgrabovsky** [comment]
> * 1 for Module imports itself

Hm, is this a false positive? I couldn't find the loop.

**Comment by ernestask** [comment]
I imagine it’s the addition of
```python
from . import util
```

**Comment by mkutlak** [comment]
>     * 1 for Module imports itself

"Resolved"

Jenkins build failure is caused by zstd. (it is not supported on RHEL-8, yet?).

**Comment by ernestask** [comment]
I’m shuffling things around in Jenkins, sorry.

**Comment by ernestask** [comment]
test this please

**Comment by ernestask** [comment]
test this please

**Comment by ernestask** [comment]
test this please

**Comment by ernestask** [comment]
retest this please

**Comment by mgrabovsky** [comment]
Is this OK to go now?

**Comment by ernestask** [comment]
You can review and merge as you see fit.

**Comment by ernestask** [comment]
I’ll figure out the story for upstream-friendly checks later, so there is no RHEL crap potentially blocking work.

File: src/checkpackage.wsgi [hunk 1]
```diff
@@ -2,11 +2,9 @@
 from webob import Request
 
 from retrace.retrace import (INPUT_ARCH_PARSER,
-                             INPUT_PACKAGE_PARSER,
```
Comment (mgrabovsky) [inline src/checkpackage.wsgi]: `INPUT_PACKAGE_PARSER` is used below on line 20. Also, you moved these parsers into `util`.

**Pull request merged**